plugins/builds/get.js

```diff
@@ -13,6 +13,15 @@ module.exports = () => ({
         description: 'Get a single build',
         notes: 'Returns a build record',
         tags: ['api', 'builds'],
+        auth: {
```

Comment:
is there a way in `hapi` to push this to a higher middleware, adding to each and every old and newer routes is cumbersome.

Else we could defined these configs as constants in a separate file and import it here.
```
auth: USER_BUILD_TOKEN/ BUILD_TOKEN/USER_TOKEN
```

Reply:
Ehh, it's technically possible via [server.auth.default(options)](https://hapijs.com/api#-serverauthdefaultoptions).  But what would you really set it to?  Because our APIs have a really mixed permission:
 - Some allow only a user to access
 - Some allow only a build to access
 - Some allow both to access
 - Some don't allow guests to access
 - Some don't have any auth (status, stats, etc.)
 - Some have custom auth (webhook, oauth, etc.)